app/src/main/res/layout/activity_create_event.xml

```diff
@@ -111,6 +111,91 @@
+                    android:layout_margin="@dimen/material_default_padding"
+                    android:layout_marginTop="0dp"
+                    android:paddingTop="0dp">
+
+                    <RadioGroup
+                        android:id="@+id/radioGroup2"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content">
+
+                        <RadioButton
+                            android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+                            android:id="@+id/endAfterRadioBtn"
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:hint="@string/ends_after"
+                            android:textColorHint="@color/black"
+                            android:checked="true"/>
+
+                        <RadioButton
+                            android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+                            android:id="@+id/endOnRadioBtn"
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:textColorHint="@color/black"
+                            android:hint="@string/ends_on"/>
+
+                    </RadioGroup>
+
+                    <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:orientation="vertical">
+                        <LinearLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="0dp"
+                            android:layout_weight="1"
+                            android:orientation="horizontal">
+                            <EditText
+                                android:id="@+id/eventRepeatsTimes"
+                                android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+                                android:textColor="@color/black"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:gravity="center_vertical"
+                                android:text="14"
+                                android:inputType="number"/>
+                            <TextView
+                                android:layout_width="wrap_content"
+                                android:layout_height="match_parent"
+                                android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+                                android:clickable="true"
+                                android:focusable="true"
+                                android:gravity="center_vertical"
+                                android:textColor="@color/black"
```

Comment:
nit: all of this hardcoding of the text color will break the "darktheme" I think

Reply:
:facepalm: You are right... how do we handle the dark mode? Is using something like color/text_dark_primary sufficient or do we need a custom style?

Reply:
About the dark mode: The standard text color is defined in resources (v27) style:
```xml
<item name="android:textColorPrimary">@color/text_primary</item>
<item name="android:textColorSecondary">@color/text_secondary</item>
```
So, if this text does not need an own fixed color, just delete this line. It will take the default colors. And else, never use a specific color directly, but define it in the colors.xml. After adding the second colors.xml for dark mode it will automatically grab the correct color depending to the settings.

Reply:
Thanks Simon! Looks a lot better now. If I have any further questions regarding this, I'll give you a call :)